bounties_api/user/backend.py

```diff
@@ -4,6 +4,9 @@
 # Best approach for now with defunct until other forms are more stable
 from eth_account.messages import defunct_hash_message
 from web3.auto import w3
+from django.conf import settings
+import datetime
```

Comment:
Double import? `from datetime import datetime, timezone` is done above

Reply:
Yeah - removed that will push it up... this is circle failing.